components/openindiana/time-slider/Makefile

```diff
@@ -21,7 +21,7 @@ COMPONENT_VERSION=	0.2.101
 COMPONENT_LICENSE=	CDDL
 COMPONENT_LICENSE_FILE=	$(COMPONENT_NAME).license
 
-COMPONENT_REVISION=$(shell cd $(COMPONENT_SRC); git rev-list HEAD --count)
+COMPONENT_REVISION=$(shell cd $(COMPONENT_SRC); git rev-list HEAD --count).1
```

Comment:
Changed timestamp is not enough to get the package updated.  If it is, then we do not need to change COMPONENT_REVISION in every package we touch.